code/ui/components/src/components/Button/Button.tsx

```diff
@@ -33,240 +125,64 @@ const ButtonWrapper = styled.button<{
     verticalAlign: 'top',
     whiteSpace: 'nowrap',
     userSelect: 'none',
-    opacity: 1,
+    opacity: disabled ? 0.5 : 1,
     margin: 0,
-    background: 'transparent',
-
-    fontSize: `${small ? theme.typography.size.s1 : theme.typography.size.s2 - 1}px`,
+    fontSize: `${theme.typography.size.s1}px`,
     fontWeight: theme.typography.weight.bold,
     lineHeight: '1',
-
-    svg: {
-      display: 'inline-block',
-      height: small ? 12 : 14,
-      width: small ? 12 : 14,
-
-      verticalAlign: 'top',
-      marginRight: small ? 4 : 6,
-      marginTop: small ? 0 : -1,
-      marginBottom: small ? 0 : -1,
-
-      /* Necessary for js mouse events to not glitch out when hovering on svgs */
```

Comment:
is this not needed anymore?

Reply:
No. I made some tests and this is not needed